kubernetes/charts/edge-kubernetes-crd/templates/crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: edgedeployments.microsoft.azure.devices.edge
+spec:
+  group: microsoft.azure.devices.edge
+  names:
+    kind: EdgeDeployment
+    listKind: EdgeDeploymentList
+    plural: edgedeployments
+    singular: edgedeployment
+  scope: Namespaced
+  version: v1
```

Comment:
Please add:
 subresources:
   # status enables the status subresource.
   status: {}